pkg/interop/native/crypto/crypto.go

```diff
@@ -46,6 +46,13 @@ func VerifyWithECDsa(msg []byte, pub interop.PublicKey, sig interop.Signature, c
 	return neogointernal.CallWithToken(Hash, "verifyWithECDsa", int(contract.NoneFlag), msg, pub, sig, curveHash).(bool)
 }
 
+// VerifyWithEd25519 calls `verifyWithEd25519` method of native CryptoLib contract and checks that sig is
+// a correct msg's signature for the given pub. Note that this method is available staritng from
+// [config.HFEchidna] hardfork.
+func VerifyWithEd25519(msg []byte, pub []byte, sig []byte) bool {
```

Comment:
@roman-khimov, I'm not sure if we need a separate type for Ed25519 pub/sig. They are not a hot path (comparing to `interop.Signature` and `interop.PublicKey`).

Reply:
OK for now.